backend/app/Http/Controllers/UsuarioController.php

```diff
@@ -103,5 +103,38 @@ public function destroy(Request $request, string $id){
         }
     }
 
+    public function checkLogin(Request $request)
+    {
+        $request->validate([
+            'cpf' => ['required'],
+            'senha' => ['required'],
+        ]);
+        
+        $usuarios = Usuario::all();
```

Comment:
No lugar de consultar todos os usuários e iterar em um for, porque não consulta apenas o usuário espécifico informado e verifica o retorno ? Isso custa menos processamento